server/webdriver/cdc_tests/autopush_test.py

```diff
@@ -40,14 +44,24 @@ def tearDown(self):
 
   def test_homepage_load(self):
     """Tests that the base autopush URL loads successfully."""
-    self.driver.get(self.url_)
-    url = self.driver.current_url
-    if not url.lower().startswith('http'):
-      raise ValueError(f'Invalid scheme in {url}. Expected http(s)://.')
-    req = urllib.request.Request(url)
-
-    # Disable Bandit security check 310. http scheme is already checked above.
-    # Codacity still calls out the error so disable the check.
-    # https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b310-urllib-urlopen
-    with urllib.request.urlopen(req) as response:  # nosec B310
-      self.assertEqual(response.getcode(), 200)
+    self.driver.get(self._base_url)
+
+    homepage_elem = find_elem(self.driver, By.ID, 'homepage')
+    self.assertIsNotNone(homepage_elem, 'Homepage element not found')
+
+  def test_statvar_explorer(self):
+    self.driver.get(f'{self._base_url}/tools/statvar')
+
+    svg_container_elem = find_elem(self.driver, By.CLASS_NAME,
+                                   'stat-var-hierarchy-container')
+    self.assertIsNotNone(svg_container_elem, 'SVG container element not found')
+
+    custom_svg_elem = wait_elem(svg_container_elem, By.XPATH,
+                                '//*[contains(text(), "OECD")]')
+    self.assertIsNotNone(custom_svg_elem, 'Custom SVG (OECD) element not found')
+
+    custom_svg_elem.click()
```

Comment:
Optional: Add a comment either inline here or on the function def that sums up the test, e.g. `"""Tests that clicking a custom SVG in the SV explorer works as expected"""`.

Reply:
Done.